elyra/metadata/storage.py

```diff
@@ -280,11 +280,11 @@ def metadata_path(*subdirs):
         # then system, where shared files will reside
         # Note, we're using getattr for these, since tests adjust the value of these
         # and we need to pull them at runtime, rather than during load.
-        system_path = getattr(jupyter_core.paths, 'SYSTEM_JUPYTER_PATH')
+        system_path = jupyter_core.paths.SYSTEM_JUPYTER_PATH
         paths.extend(system_path)
```

Comment:
This was necessary at one point for testing, but I suspect that necessity was removed during the refactor in #708. Since the tests are passing, I don't think its worthwhile to dig that up.

Could you please remove comment beginning with `# Note, we're using getattr...`?